### PR TITLE
Added annotation for GeoPosition

### PR DESCRIPTION
**Pull request by ue71603**

### CODE REVIEW

**Review by trurlurl — APPROVED** [review]

File: OJP/OJP_PlaceSupport.xsd [hunk 1]
```diff
@@ -456,7 +456,11 @@
 					<xs:documentation>Public name of the place.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
-			<xs:element name="GeoPosition" type="siri:LocationStructure"/>
+			<xs:element name="GeoPosition" type="siri:LocationStructure">
+				<xs:annotation>
+					<xs:documentation>WGS84 coordinates position.</xs:documentation>
```
Comment (skinkie) [inline OJP/OJP_PlaceSupport.xsd:461]:
```suggestion
					<xs:documentation>Position using WGS84 coordinates.</xs:documentation>
```

**Review by skinkie — APPROVED** [review]

**Review by sgrossberndt — DISMISSED** [review]

**Review by skinkie — APPROVED** [review]

**Comment by skinkie** [comment]
@sgrossberndt was there now an option to do this without merging again?

**Comment by sgrossberndt** [comment]
not yet

**Review by sgrossberndt — APPROVED** [review]

**Review by trurlurl — COMMENTED** [review]
What do we have the OJP GeoPositionStructure for? If GeoPositionStructure were redundant with siri:LocationStructure, I would find it nice to always use the same of the two.

**Comment by trurlurl** [comment]
> @trurlurl Where did you find that structure? In my view it does not exist.

@ue71603 Your're right - it doesn't exist any more (used to be in OJP_Utility.xsd). So my comment is irrelevant / resolved.

**Review by trurlurl — APPROVED** [review]

**Review by skinkie — APPROVED** [review]

**Pull request merged**